plugin/files/airconsole/runtime.js

```diff
@@ -86,6 +88,7 @@ cr.plugins_.AirConsole = function(runtime)
       if (!device_data) {
         self.ac_leave_id = device_id;
         self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnDeviceLeft, self);
+		self.runtime.trigger(cr.plugins_.AirConsole.prototype.cnds.OnAnyDeviceLeft, self);
```

Comment:
Sure, will do. Didn't want to push the highscores, I have no way to test them really. Communication works, triggers do their job, but I don't know about the data received